layers/meta-balena-fsl-arm/recipes-bsp/imx-mkimage/imx-mkimage%.bbappend

```diff
@@ -1,3 +1,3 @@
 # Use latest and known to work revision. Otherwise if
 # booting fails at some point we won't know what caused it
-SRCREV = "2bb49a37e1d964eacebb09aa0073048a2bfd26e8"
+SRCREV = "dc22e26d07f249325604f7dfd42324a9a75e7cc0"
```

Comment:
Same question about autorev as for the bootloader

Reply:
We're using the master branch, where all these are autorev.